lib/charms/traefik_k8s/v0/ingress_per_leader.py

```diff
@@ -28,26 +29,26 @@
 Then, to initialise the library:
 
 ```python
-from charms.traefik_k8s.v1.ingress import (IngressPerAppRequirer,
-  IngressPerAppReadyEvent, IngressPerAppRevokedEvent)
+from charms.traefik_k8s.v2.ingress import (IngressPerLeaderRequirer,
```

Comment:
Mismatch with folder name. Not sure if we should start from 0 though.
```suggestion
from charms.traefik_k8s.v0.ingress import (IngressPerLeaderRequirer,
```

Reply:
umh, not sure what's going on. On local it's in v2.

Reply:
ah, my bad. It's in a comment. It should be `v0.ingress_per_leader`. Fixed in next commit